src/jest/config.ts

```diff
@@ -20,15 +22,19 @@ export async function getDappeteerConfig(): Promise<DappateerJestConfig> {
       metaMask: {},
     };
 
-  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-unsafe-assignment
   const config = await require(filePath);
 
   return {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     dappeteer: {
       ...DAPPETEER_DEFAULT_CONFIG,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
       ...config.dappeteer,
     },
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     metaMask: {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
       ...config.metaMask,
     },
   };
```

Comment:
I think it will be better to wrap in the `eslint-disable`  block

```suggestion
  /* eslint-disable @typescript-eslint/no-unsafe-assignment */
  return {
    dappeteer: {
      ...DAPPETEER_DEFAULT_CONFIG,
      ...config.dappeteer,
    },
    metaMask: {
      ...config.metaMask,
    },
  };
  /* eslint-enable @typescript-eslint/no-unsafe-assignment */
```